limitador-server/src/main.rs

```diff
@@ -588,6 +589,18 @@ fn create_config() -> (Configuration, &'static str) {
                 .about("Uses Redis to store counters, with an in-memory cache")
                 .display_order(4)
                 .arg(redis_url_arg)
+                .arg(
+                    Arg::new("batch")
+                        .long("batch-size")
+                        .action(ArgAction::Set)
+                        .value_parser(clap::value_parser!(u64))
+                        .default_value(
+                            config::env::REDIS_LOCAL_CACHE_BATCH_SIZE
+                                .unwrap_or(leak(DEFAULT_BATCH_SIZE)),
+                        )
+                        .display_order(3)
```

Comment:
I actually don't know if it matters or why it was starting from 4 before, but shouldn't these sub-options start at 1 through 4.. although I guess as long as they can be ordered it doesn't matter

Reply:
It's "just" a way to sort the options, by avoiding changing the order... e.g. to avoid crappy diffs. Which is why I didn't change these here neither and keep the diff to the "useful" stuff